content/browser-automation.md

```diff
@@ -1,17 +1,11 @@
 ---
 menu: reference
 renderer: Cucumber::Website::Reference
```

Comment:
Remove 'renderer:' as I don't think it's used anymore?

Reply:
I don't think so if we're listing tools like this - best to show people what there is. We should however make it more clear which are language specific